### PR TITLE
Feature/modify base manifest sample

### PR DESCRIPTION
**Pull request by okmtz**
### Description

- modify sample `config/base_manifest.yaml` parallelism field value.
- add explanation about type matching between `config/base_manifest.yaml` and Gatling struct field in related docs.

### Checklist

_Please check if applicable_

- [x] Tests have been added (if applicable, ie. when cli codes are added or modified)
- [x] Relevant docs have been added or modified (if applicable, ie. when new features are added or current features are modified)

### CODE REVIEW

File: docs/user-guide.md [hunk 1]
```diff
@@ -27,8 +27,13 @@ Configuration values for the load test are written in `config/config.yaml`.
 
 The `base_manifest.yaml` describes the common values for each load test in the Kubernetes manifest of the Gatling Resource.
 
-Fields marked `<config.yaml overrides this field>` in `base_manifest.yaml` are set to different values for each load test. The value of this field will be replaced by the value in `config.yaml` respectively when Gatling Commander runs.
-Therefore, setting values to fields marked `<config.yaml overrides this field>` in `base_manifest.yaml` is not necessary.
+Fields marked `<config.yaml overrides this field>` in `base_manifest.yaml` are set to different values for each load test. The value of this field will be replaced by the value in `config.yaml` respectively when Gatling Commander runs. Therefore, setting values to fields marked `<config.yaml overrides this field>` in `base_manifest.yaml` is not necessary.
+
+\* Gatling Commander once loads `base_manifest.yaml` value to Gatling struct object before replaced by `config.yaml`. So the type of `base_manifest.yaml` field vlaue must be matched to Gatling struct field one. If type not match, like following error will be occured.
```
Comment (kayamin) [inline docs/user-guide.md]: @okmtz 
【IMO】
Corrected sentences.
```suggestion
\* Gatling Commander loads `base_manifest.yaml` value to Gatling struct object before it replaces the value by `config.yaml`. So the type of `base_manifest.yaml` field value must be matched to Gatling struct field one. If type not match, an error like following will occur.
```

Comment (okmtz) [reply docs/user-guide.md]: @kayamin 
commit suggestion and rebase with my old commit !
thank you !

File: docs/user-guide.jp.md [hunk 2]
```diff
@@ -96,9 +96,14 @@ serviceは同一の負荷試験対象に関する1つ以上の負荷試験シナ
 `base_manifest.yaml`にはGatlingリソースのKubernetesマニフェストのうち、負荷試験ごとに共通する値を設定するフィールドを記述します。  
 GatlingリソースのKubernetesマニフェストのフィールドについては、[Gatling OperatorのAPI Reference](https://github.com/st-tech/gatling-operator/blob/main/docs/api.md#gatling)を参照してください。
 
-`base_manifest.yaml`に`<config.yaml overrides this field>`と記載があるフィールドは、負荷試験ごとに異なる値が設定されます。  
-これらのフィールドの値は、Gatling Commanderの実行時に`config.yaml`の値でそれぞれ置き換えられます。  
-そのため、`base_manifest.yaml`での値の変更は不要です。
+`base_manifest.yaml`に`<config.yaml overrides this field>`と記載があるフィールドは、負荷試験ごとに異なる値が設定されます。これらのフィールドの値は、Gatling Commanderの実行時に`config.yaml`の値でそれぞれ置き換えられます。そのため、`base_manifest.yaml`での値の変更は不要です。
```
Comment (kenz) [inline docs/user-guide.jp.md:99]: 【IMO】 `base_manifest.yaml`による値の変更がいつの話か、どのようなメリットがあるかを明示するとよさそう。
```suggestion
`base_manifest.yaml`に`<config.yaml overrides this field>`と記載があるフィールドは、負荷試験ごとに`config.yaml`の値で上書きされます。そのため、`base_manifest.yaml`の値を変更する事なく複数の負荷試験を連続して実行できます。```

File: docs/user-guide.md [hunk 3]
```diff
@@ -27,8 +27,13 @@ Configuration values for the load test are written in `config/config.yaml`.
 
 The `base_manifest.yaml` describes the common values for each load test in the Kubernetes manifest of the Gatling Resource.
 
-Fields marked `<config.yaml overrides this field>` in `base_manifest.yaml` are set to different values for each load test. The value of this field will be replaced by the value in `config.yaml` respectively when Gatling Commander runs.
-Therefore, setting values to fields marked `<config.yaml overrides this field>` in `base_manifest.yaml` is not necessary.
+Fields marked `<config.yaml overrides this field>` in `base_manifest.yaml` are set to different values for each load test. The value of this field will be replaced by the value in `config.yaml` respectively when Gatling Commander runs. Therefore, setting values to fields marked `<config.yaml overrides this field>` in `base_manifest.yaml` is not necessary.
+
+\* Gatling Commander once loads `base_manifest.yaml` value to Gatling struct object before it replaces the value by `config.yaml`. So the type of `base_manifest.yaml` field value must be matched to Gatling struct field one. If type not match, an error like following error will be occured.
```
Comment (kayamin) [inline docs/user-guide.md]: 【nits】
```suggestion
\* Gatling Commander once loads `base_manifest.yaml` value to Gatling struct object before it replaces the value by `config.yaml`. So the type of `base_manifest.yaml` field value must be matched to Gatling struct field one. If type not match, an error like following occur.
```

**Review by kayamin — APPROVED** [review]
LGTM!

**Review by kenz — COMMENTED** [review]
一点だけコメントしました

File: docs/user-guide.jp.md (hunk 2 above)
Comment (okmtz) [reply docs/user-guide.jp.md:99]: 別PRに分けて修正します！

**Pull request merged**